src/Classes/Board.js

```diff
@@ -16,11 +16,50 @@ import Mandrake from "./plants/Mandrake";
 import Player from "./Animal/Player";
 let map = document.querySelector('.map');
 
+
+
+
 export default class Board {
     grid = null;
     player = null;
     constructor() {
         this.initialize();
+        this.upLeftButton = document.querySelector(".up-left-button");
```

Comment:
There is quite a lot of logic here. We could fit in a separate `Controls` class.
It could accept the `this.movePlayer` as the `onMove` argument.

```
class Controls {
  constructor(onMove, player) {
    // ...
  }
}
```

```
class Board {
  controls = new Controls(this.movePlayer, this.player)
  // ...
}
```